lib/ecto/changeset.ex

```diff
@@ -147,8 +148,8 @@ defmodule Ecto.Changeset do
 
   ## Associations, embeds, and on replace
 
-  Using changesets you can work with associations as well as with embedded
-  structs. There are two primary APIs:
+  Using changesets you can work with associations as well as with
+  [embedded](embedded-schemas.html) structs. There are two primary APIs:
```

Comment:
💯

Reply:
This link was suggested to me by someone and it helps a lot. The problem I suspect is it’s in another tab in the menu. I spent ten minutes this morning trying to decide if there was another way to organize those to make the guides more obvious to someone coming in via a Google search, and… I’ve got nothing that isn’t worse than the status quo. But I might double check the media queries.  I think I was running into horizontal scrollbars on desktop and I need to doublecheck.

Reply:
Adding more hyper links is good. If you are new to a library, there is so much you can learn, and you don't have the complete understand of where to go next and how things relate, the links help with that. :+1:

Reply:
I still haven't figured out the bug that brought me here in the first place.